viewer/fetch_from_cedar.sh

```diff
@@ -12,7 +12,7 @@ SOURCEGLOB="rev${REV}_*.html"
 SLACK_WEBHOOK=/usr/local/share/slack/daily_pipeline_webhook
 VIEWER_LINK="https://bao.chimenet.ca/daily/view?csd="
 
-DESTBASE=/mnt/md1/daily
+DESTBASE=/opt/venvs/daily_viewer/viewer
```

Comment:
Copying the rendered notebooks into the `venv` dir seems like an odd choice to me. It seems better just to share a single global data directory, but that seems to be the previous scheme you've abandoned.

Reply:
Sure, we can do that.  I've moved this script to ch_ansible so it can be templated, meaning how we set things up can be more flexible.